docs/modules/storage/pages/pooled-native-memory.adoc

```diff
@@ -0,0 +1,51 @@
+= Pooled Native Memory
```

Comment:
"POOLED native memory" -> "POOLED native memory allocator" in a few places (text, title, page name). "pooled" is a kind of allocator, not a kind of memory